tests/Notificator.Redis.test.js

```diff
@@ -0,0 +1,48 @@
+const { redisPubsub } = require("../etc/config");
+const Notificator = require("../lib/Notificator");
+const notificator = require("../lib/notificatorSingletonRedis");
+const PubSub = require("../lib/PubSub");
+const RedisDriver = require("../lib/drivers/Redis");
+
+describe("Notificator with Redis driver tests", () => {
+  // test("test placeholder", async () => {});
+  test("positive: can notify/receive messages", async () => {
+    await notificator.init();
+    setTimeout(() => {
+      notificator.notify({ test: 1 });
+    }, 100);
+    const result = await new Promise((res) =>
+      notificator.receive((msg) => res(msg))
+    );
+    expect(result.test).toBe(1);
+  });
+  // test("positive: can notify/receive messages between different nodes", async () => {
+  //   const notificatorOne = createNotificator();
+  //   const notificatorTwo = createNotificator();
+  //   await notificatorOne.init();
+  //   await notificatorTwo.init();
+  //   setTimeout(() => {
+  //     notificatorOne.notify({ test: 1 });
+  //   }, 100);
+  //   const result = await new Promise((res) =>
+  //     notificatorTwo.receive((msg) => res(msg))
+  //   );
+  //   expect(result.test).toBe(1);
+  // });
+});
+
+function createNotificator() {
+  const rabbitDriver = new RedisDriver({
```

Comment:
rabbitDriver => redisDriver

Reply:
Thank you for your attention to details :)

Reply:
Fixed in 55de5d98157be8ecfa63d9756f9ea5baaf277544.